activerecord/test/cases/base_test.rb

```diff
@@ -968,6 +968,14 @@ def test_previously_new_record_returns_boolean
     assert_equal false, Topic.find(1).previously_new_record?
   end
 
+  def test_previously_new_record_on_destroyed_record
+    topic = Topic.create
+    assert topic.previously_new_record?
```

Comment:
Nitpick but generally `assert_predicate` provides a better error message. Even though we may never see this test fail in case if it fails `assert_predicate` will produce something like "expected `Topic<id:123>` to be :previously_new_record?" which allows for better understanding of the failure while `assert` will produce a very dry "expected false to be truthy" message 

```suggestion
    assert_predicate topic, :previously_new_record?
```

Reply:
Agreed, thanks Nikita!